test/models/solution_test.rb

```diff
@@ -30,4 +30,17 @@ class SolutionTest < ActiveSupport::TestCase
 
     refute solution.auto_approve?
   end
+
+  test "active_mentors" do
+    solution = create :solution
+    user1 = create :user
+    user2 = create :user
+
+    create :solution_mentorship, solution: solution
```

Comment:
What does creating this solution mentorship do? It adds a user in the background, right?